pkgs/development/python-modules/b2sdk/default.nix

```diff
@@ -47,7 +48,8 @@ buildPythonPackage rec {
     substituteInPlace setup.py \
       --replace 'setuptools_scm<6.0' 'setuptools_scm'
     substituteInPlace requirements.txt \
-      --replace 'arrow>=0.8.0,<1.0.0' 'arrow'
+      --replace 'arrow>=0.8.0,<1.0.0' 'arrow' \
```

Comment:
we should probably remove the no-longer-relevant entry
```suggestion
```

Reply:
It's still present too, just applies for older python versions.